src/Databases/MySQL/FetchTablesColumnsList.cpp

```diff
@@ -94,21 +96,24 @@ std::map<String, NamesAndTypesList> fetchTablesColumnsList(
         const auto & char_max_length_col = *block.getByPosition(5).column;
         const auto & precision_col = *block.getByPosition(6).column;
         const auto & scale_col = *block.getByPosition(7).column;
+        const auto & column_comment_col = *block.getByPosition(8).column;
 
         size_t rows = block.rows();
         for (size_t i = 0; i < rows; ++i)
         {
             String table_name = table_name_col[i].safeGet<String>();
-            tables_and_columns[table_name].emplace_back(
-                    column_name_col[i].safeGet<String>(),
-                    convertMySQLDataType(
-                            type_support,
-                            column_type_col[i].safeGet<String>(),
-                            settings.external_table_functions_use_nulls && is_nullable_col[i].safeGet<UInt64>(),
-                            is_unsigned_col[i].safeGet<UInt64>(),
-                            char_max_length_col[i].safeGet<UInt64>(),
-                            precision_col[i].safeGet<UInt64>(),
-                            scale_col[i].safeGet<UInt64>()));
+            tables_and_columns[table_name].add(
```

Comment:
Did you consciously change from emplace_back to add here? Not performance critical, so I guess you could argue readability (add) wins over performance (emplace_back) in this case.

Reply:
It's just that `ColumnsDescription` has no `emplace_back` method.